### ISSUE TITLE
"OP SAFT"

### ISSUE THREAD

**Issue by HazardJ** [BranDAOn/OP-SAFT#3]
As far as I can tell, the term "OP SAFT" is used for the instrument. but is not a defined term.  The term "SAFT" is defined.  Is this intended?

**Comment by HazardJ** [comment]
Whoops, the term "OP SAFT" is quoted (and therefore defined?) in the title of the instrument.

**Comment by BranDAOn** [comment]
That's right, I think. There aren't any disclaimers for headings not being operative / binding, and the pseudo-preamble provides for all "undefined capitalized terms" to be defined with reference to the annex, so theoretically including the defined term in the annex should be enough for a court to find that the parties intended for the term "OP SAFT" to refer to https://github.com/BranDAOn/OP-SAFT/blob/main-(non-U.S.-investor)/FORM-OF-OP-SAFT.md (or the document) generally.
